docs/ref/models/database-functions.txt

```diff
@@ -804,6 +851,25 @@ Usage example::
     >>> print(Author.objects.get(name='Margaret Smith').alias)
     marga
 
+``Trim``
+--------
+
+.. class:: Trim(expression, **extra)
+
+.. versionadded:: 2.1
+
+Accepts a single text field or expression and returns a string after removing
+space characters from its both ends.
```

Comment:
Would it be worth mentioning that Python `str.strip` removes different characters? This one only removes space characters.

Reply:
I don't think we need any extra explanation. IMO the doc is clear: _"...removing space characters..."_.